### PR TITLE
Add last modified to batches

### PR DESCRIPTION
**Pull request by xrigau**
Adds a 'last modified' column to the batches table

The column is updated with `System.currentTimeMillis()` when a new row is created and every time a row is updated

I did some boyscouting too!

### CODE REVIEW

File: library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java [hunk 1]
```diff
@@ -174,7 +174,13 @@
      * Timestamp when the download was last modified, in {@link System#currentTimeMillis
      * System.currentTimeMillis()} (wall clock time in UTC).
      */
-    public static final String COLUMN_LAST_MODIFIED_TIMESTAMP = "last_modified_timestamp";
+    public static final String COLUMN_DOWNLOAD_LAST_MODIFIED_TIMESTAMP = "download_last_modified_timestamp";
```
Comment (xrigau) [inline library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java]: wasn't sure about this maybe this one shouldn't be exposed publicly

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/download-manager-prb/115/
Test FAILed.

**Comment by amlcurran** [comment]
what is it used for?

**Comment by ouchadam** [comment]
what @amlcurran said

**Pull request closed without merging**